ci_framework/hooks/playbooks/restart_nova_scheduler.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Perform CRC login
+      ansible.builtin.include_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+
+    - name: Restart nova-scheduler to pick up cell1
+      ansible.builtin.command: >-
+        oc delete pod -n {{ cifmw_install_yamls_defaults['NAMESPACE'] | default('openstack') }} -l service=nova-scheduler
```

Comment:
No need to default is, this variable should be always there, as is exported by the install_yamls role filled with the default value of the Makefile and, if the environment has been overridden with edpm_env, the value in the dict will be the overridden too.